### PR TITLE
[docs] Getting Started: Introduction and Installation

### PR DESCRIPTION
**Pull request by hogepodge**
The first two sections of the "Getting Started with TVM" guide.
A high level introduction to TVM, and a slight introduction
on installation options.

Co-authored-by: Jocelyn Shiue <jshiue@octoml.ai>

### CODE REVIEW

**Comment by hogepodge** [comment]
@tqchen

File: tutorials/get_started/install.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# about TLCPack at their `website <https://tlcpack.ai>`_. TLCPack offers a
+# support matrix with instructions for how to install TVM on different
+# platforms, with different features. If installing with pip or conda through
+# TLCPack, we recommend to use one of the nightly builds, as they contain the
+# latest version of `tvmc`. Some platforms and configurations are not
+# available.  If your environment isn't supported, you can install TVM from
+# source.
+
+################################################################################
+# Installing from Source
```
Comment (tqchen) [inline tutorials/get_started/install.py:45]: move source to the beginning as officially we release only the source pkg

**Review by tqchen — CHANGES_REQUESTED** [review]

**Comment by hogepodge** [comment]
@tqchen ready for review

File: tutorials/get_started/install.py (hunk 1 above)
Comment (hogepodge) [reply tutorials/get_started/install.py]: done

**Review by tqchen — APPROVED** [review]

**Pull request merged**